binary_transparency/firmware/api/update_package.go

```diff
@@ -19,7 +19,7 @@ type UpdatePackage struct {
 	// FirmwareImage is the actual firmware image itself.
 	FirmwareImage []byte
 	// ProofBundle holds the various artifacts required to validate the firmware image.
```

Comment:
This comment should be extended to note the format of this byte slice (JSON).

Reply:
ta, done